terracumber-cli

```diff
@@ -67,11 +67,16 @@ def parse_args():
                                              either environment variable BUILD_NUMBER from Jenkins
                                              or BUILD_TIMESTAMP variable (manually managed) must be
                                              exported""",
-                        choices=['gitsync', 'provision', 'cucumber', 'getresults', 'mail'], default=False)
+                        choices=['gitsync', 'provision', 'cucumber', 'getresults', 'mail',
```

Comment:
As now this is not just about sumaform, we should probably briefly adapt the markdown files to document the new stuff.

Reply:
Oh, true. Makes totally sense.

Reply:
Tests added and markdown pages adjusted.